recipes-core/varstored/varstored_git.bb

```diff
@@ -0,0 +1,76 @@
+SUMMARY = "Package for managing guest EFI variables"
+LICENSE = "BSD"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=8444b396c3cde7d8fe18ae36a3638a55"
+
+inherit useradd xc-rpcgen-c
+
+DEPENDS = " \
+    dbus \
+    dbus-glib \
+    libseccomp \
+    libxml2 \
+    libxcdbus \
+    openssl \
+    xen-tools \
+"
+
+CFLAGS_prepend += "-I${STAGING_INCDIR}/dbus-1.0 -I${STAGING_LIBDIR}/dbus-1.0/include -I${STAGING_INCDIR}/glib-2.0 -I${STAGING_LIBDIR}/glib-2.0/include "
+LDFLAGS_prepend += " -levent -lgobject-2.0 -lglib-2.0 -ldbus-glib-1 -lxcdbus "
```

Comment:
These lines look wrong.  Seems like the Makefile should be patched to add them.

Reply:
Hm can we dynamically find the headers in the staging dir if we move to the Makefile? I thought autotools were needed for that kind of thing in OE. That's why I went with this approach.

Reply:
For CFLAGS, I'm not sure how you could add a sysroot-prefixed directory to the includes.  But you can sidestep that issue by expanding relevant include lines to add the prefix, like dbus-1.0/dbus/dbus.h

LDFLAGS additions should be in the Makefile